ribs/emitters/_emitter_base.py

```diff
@@ -99,14 +98,18 @@ def upper_bounds(self):
     def ask(self):
         """Generates an ``(n, solution_dim)`` array of solutions."""
 
-    def tell(self, solutions, objective_values, behavior_values, metadata=None):
+    @abstractmethod
+    def tell(self,
+             solutions,
```

Comment:
Use the batch terminology here with `solution_batch`, `objective_batch`, `measures_batch`, `metadata_batch`